scripts/analysis.py

```diff
@@ -105,13 +105,13 @@ def get_intercept(x, y, m, b):
 print(f"Y Velocity Random Walk: {accel_wn_intercept_y: .5f} m/s/sqrt(s) {accel_wn_intercept_y*60: .5f} m/s/sqrt(hr)")
 print(f"Z Velocity Random Walk: {accel_wn_intercept_z: .5f} m/s/sqrt(s) {accel_wn_intercept_z*60: .5f} m/s/sqrt(hr)")
 
-print(f"X Bias Instability: {accel_min_x: .5f} m/s^2 {accel_min_x*60*60: .5f} m/hr^2")
-print(f"Y Bias Instability: {accel_min_y: .5f} m/s^2 {accel_min_y*60*60: .5f} m/hr^2")
-print(f"Z Bias Instability: {accel_min_z: .5f} m/s^2 {accel_min_z*60*60: .5f} m/hr^2")
+print(f"X Bias Instability: {accel_min_x: .5f} m/s^2 {accel_min_x*60*60: .5f} m/s/hr")
+print(f"Y Bias Instability: {accel_min_y: .5f} m/s^2 {accel_min_y*60*60: .5f} m/s/hr")
+print(f"Z Bias Instability: {accel_min_z: .5f} m/s^2 {accel_min_z*60*60: .5f} m/s/hr")
 
-print(f"X Accel Random Walk: {accel_rr_intercept_x: .5f} m/s^2")
```

Comment:
Why should these units be changed? Are you perhaps trying to match the units in the Kalibr wiki? There's a known mistake in the units there as discussed here: https://github.com/ori-drs/allan_variance_ros/issues/2

Reply:
I'll reply more in depth in a separate comment. These inline comments tend to get lost when rebasing.